frontend/public/components/operator-lifecycle-manager/create-operand.tsx

```diff
@@ -331,7 +332,7 @@ export const CreateOperandForm: React.FC<CreateOperandFormProps> = (props) => {
     }
     if (field.capabilities.includes(SpecCapability.booleanSwitch)) {
       return <AsyncComponent
-        loader={() => import('patternfly-react').then(m => m.Switch)}
+        loader={() => Switch}
```

Comment:
No reason to use `AsyncComponent` now. Are we certain this change is needed?

We might just switch to the PF4 switch.

Reply:
It's needed or we need to make a Webpack chunk for it so all of `patternfly-react` isn't included.

Reply:
> It's needed or we need to make a Webpack chunk for it so all of `patternfly-react` isn't included.

Right, I see it now 👍